### PR TITLE
Slightly adjust beta tag on course tall block to look better.

### PR DESCRIPTION
**Pull request by breville**
Good suggestion from @laurelfan to go from this:

screenshot 2015-11-23 12 50 24

to

screenshot 2015-11-23 13 01 54

### CODE REVIEW

**Comment by breville** [comment]
fyi @tanyaparker

File: shared/css/course-blocks.scss [hunk 1]
```diff
@@ -112,13 +112,14 @@
...
 
-    h3 {
+    .title {
       width: 135px;
       padding-left: 10px;
       float: left;
     }
 
     .betatext {
+      color: $realyellow;;
```
Comment (tanyaparker) [inline shared/css/course-blocks.scss]: LOL @realyellow

extra semi-colon doe

**Comment by tanyaparker** [comment]
Cool beans! :+1:

**Comment by laurelfan** [comment]
LGTM

**Pull request merged**